src/array.js

```diff
@@ -59,7 +63,7 @@ function getArrayOfPositives(arr) {
  *    [ false, 0, NaN, '', undefined ]   => [ ]
  */
 function removeFalsyValues(arr) {
-	throw new Error("Not implemented");
+	return arr.filter(elem=> (elem!=null && elem!=undefined && elem!=false && elem!='' && (""+elem)!="NaN" && elem!=0))
```

Comment:
You don't need to check soo many conditions. JS treats all these values as `falsy`, i.e. equivalent to false. Try to read about it online.